exampleSite/content/samples/icons/index.md

```diff
@@ -40,6 +40,7 @@ The full list of built-in icons and their corresponding names can referenced bel
 | gitlab               | {{< icon gitlab >}}               |
 | globe                | {{< icon globe >}}                |
 | google               | {{< icon google >}}               |
+| google scholar       | {{< icon google-scholar >}}       |
```

Comment:
This should be `google-scholar` as the icon name in the docs needs to include the hyphen so that users know to use it when referring to the icon